frontend/packages/console-dynamic-plugin-sdk/src/api/dynamic-core-api.ts

```diff
@@ -128,3 +121,24 @@ export const InventoryItemLoading: React.FC = require('@console/shared/src/compo
   .InventoryItemLoading;
 
 export { useFlag } from '../utils/flags';
+
+/**
+ * A lazy component that creates a monaco editor
+ * @example
+ * ```tsx
+ * <DroppableEditYAML
+ *     obj={obj}
+ *     create
+ *     resourceObjPath={resourceObjPath}
+ *     header={`Create ${kindObj.label}`}
+ *    />
+ * ```
+ * @param {string} obj - Stringified YAML to be shown by the Editor
+ * @param {boolean} create - Enable creation of resource
+ * @param {DroppableEditYAMLProps['resourceObjPath']} resourceObjPath - A function to generate the redirect path for resource creations
+ * @param {string} header - Add a custom header for the YAML editor
+ * @param {DroppableEditYAMLProps['onSave']} - A function that handles save requests
+ *
+ */
+export const DroppableEditYAML: React.FC<DroppableEditYAMLProps> = require('@console/internal/components/AsyncYamlEditor')
```

Comment:
`DroppableEditYAML` doesn't seem like a great name. Maybe `ResourceYAMLEditor`?

Reply:
I think `DroppableEditYAML` was named like that because you can drag and drop files into it